src/isotp_sender.cpp

```diff
@@ -142,13 +142,20 @@ int IsoTpSender::sendData(const void* buffer, size_t size) const noexcept
         return -1;
     }
 
-    auto bytes_sent = write(send_skt_, buffer, size);
-    if (bytes_sent < 0)
+    int bytes_sent = 0;
+    int retries = 5;
```

Comment:
Maybe turn that magic number into a constant

Reply:
done